packages/core/src/lib/toast/toast-banner.svelte

```diff
@@ -33,9 +33,9 @@ export { extraClasses as cx };
       aria-hidden={false}
       aria-label="success"
     />
-  </div>
 
-  <div class="mr-auto">{message}</div>
+    {message}
+  </div>
 
   {#if action}
     <button
```

Comment:
Missing `type="button"` on this button, which is important because buttons without an explicit `type` can change out from under you, e.g. by implicitly becoming a `type="submit"` if inside a form